07-routing-gossip.md

```diff
@@ -524,6 +524,13 @@ makes sense to have it be a UNIX timestamp (i.e. seconds since UTC
 1970-01-01). This cannot be a hard requirement, however, given the possible case
 of two `channel_update`s within a single second.
 
+It is assumed that more than one `channel_update` message changing the channel 
+parameters in the same second is a DoS attempt, and therefore, the node responsible 
```

Comment:
I'd be less strict on categorizing something as a DoS attempt, you'd need more historical data of a peer misbehaving before putting it in a blacklist.

```suggestion
parameters in the same second may be a DoS attempt, and therefore, the node responsible 
```